### ISSUE TITLE
Definir l'utilisation de chaque outils

### ISSUE THREAD

**Issue by xinity** [DockerLyon/community#5]
Nous avons pas d'outils dont certains pourraient se recouper comme par exemple: trello et les issues/PR de Github.

Il est important que l'on s'entends sur comment utiliser quel outils pour ne pas faire double emploi :)

**Comment by emmanuelCarre** [comment]
Yes, I am agree. We will list us tools into this repository. We can add paragraph to detail the usage of each tool.

Do you think it will be enough?
